### PR TITLE
Get AKS Service CIDR

### PR DESCRIPTION
**Pull request by ksubrmnn**
Please provide a description for what this PR is for.

Describes how to get service CIDR for AKS for egress set up
And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [ ] Docs
- [ ] Installation
- [X] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/traffic-management/egress/egress-control/index.md [hunk 1]
```diff
@@ -389,10 +389,21 @@ servicesIpv4Cidr: 10.7.240.0/20
 
 Use `--set values.global.proxy.includeIPRanges="10.4.0.0/14\,10.7.240.0/20"`
 
-#### Azure Container Service(ACS)
+#### Azure Container Service (ACS)
```
Comment (nshankar13) [inline content/en/docs/tasks/traffic-management/egress/egress-control/index.md:392]: You can get rid of ACS altogether.

File: content/en/docs/tasks/traffic-management/egress/egress-control/index.md [hunk 2]
```diff
@@ -389,9 +389,47 @@ servicesIpv4Cidr: 10.7.240.0/20
...
+$ az aks show --resource-group ${RESOURCE_GROUP} --name ${CLUSTER} | grep Cidr
+    "podCidr": "10.244.0.0/16",
+    "podCidrs": [
+    "serviceCidr": "10.0.0.0/16",
+    "serviceCidrs": [
+{{< /text >}}
+
+Then use `--set values.global.proxy.includeIPRanges="10.244.0.0/16\,10.0.0.0/16"`
+
+##### Azure CNI
```
Comment (nshankar13) [inline content/en/docs/tasks/traffic-management/egress/egress-control/index.md:408]: Just remembered, we should also add a caveat that the Azure CNI steps here are for non-overlay networking modes. With Azure CNI overlay, the steps should be the same as Kubenet for getting PodCIDR and ServiceCIDR.

**Review by nshankar13 — APPROVED** [review]

File: content/en/docs/tasks/traffic-management/egress/egress-control/snips.sh [hunk 3]
```diff
@@ -211,6 +211,41 @@ clusterIpv4Cidr: 10.4.0.0/14
...
+ENDSNIP
+
+snip_azure_cni_2() {
+az aks show --resource-group ${RESOURCE_GROUP} --name ${CLUSTER} | grep nodeResourceGroup
+}
+
+! read -r -d '' snip_azure_cni_2_out <<\ENDSNIP
+  "nodeResourceGroup": "MC_user-rg_user-cluster_eastus",
+  "nodeResourceGroupProfile": null,
+az network vnet list -g MC_user-rg_user-cluster_eastus | grep name
```
[the remark this answers is not in the document]
Comment (ksubrmnn) [reply content/en/docs/tasks/traffic-management/egress/egress-control/snips.sh]: I removed the `<>` for better formatting, but otherwise used this

**Review by dhawton — APPROVED** [review]

**Pull request merged**